.github/pull_request_template.md

```diff
@@ -8,4 +8,6 @@
  - [ ] Keymaps do not use deprecated key defines (Check using the [upgrader tool](https://zmk.dev/docs/codes/keymap-upgrader))
  - [ ] `&pro_micro` used in favor of `&pro_micro_d/a` if applicable
  - [ ] If split, no name added for the right/peripheral half
+ - [ ] Kconfig.defconfig file correctly wraps *all* configuration in conditional on the shield symbol
  - [ ] `.conf` file has optional extra features commented out
+ - [ ] Keyboard/PCB is part of a shipped group buy or is generally available in stock to purchase.
```

Comment:
Should we mention something like "OSH/personal projects without general availability should take advantage of a `zmk-config` repository instead"?

Reply:
Indeed. Updated.